action/devices.py

```diff
@@ -203,3 +205,11 @@ def add_devices(devices: str):
                 added_devices.append(new_device)
         else:
             print(f"WARNING: Device {device_name} not found")
+
+    return Section(
+        "device",
+        dependencies=[],
+        refers="target",
+        echo=True,
+        commands=[Command(command=[command]) for commands in added_devices for command in commands.add_commands]
+    )
```

Comment:
Could we have YAML sections prepared for each kind of device and then pass in device parameters as variables?